app/views/live/show_pages.html.erb

```diff
@@ -12,7 +12,7 @@
     <%= render FormStatusTagDescriptionComponent::View.new(status: :live) %>
 
     <% form.pages.each_with_index do |page, index| %>
-      <%= render(SummaryCardComponent::View.new( **PageSummaryCardDataService.call(page:).build_data)) %>
+      <%= render(SummaryCardComponent::View.new( **PageSummaryCardDataService.call(page:, pages: form.pages).build_data)) %>
```

Comment:
Should we maybe have an example of the Summary Card component displaying routing options?

Reply:
Done!